cli/ballerina-cli/src/main/java/io/ballerina/cli/utils/NativeUtils.java

```diff
@@ -82,6 +82,27 @@ public static void createReflectConfig(Path nativeConfigPath, Package currentPac
             Map<String, String> testUtilityFunctions = testSuiteMap.get(moduleName).getTestUtilityFunctions();
             //Add init class
             if (testUtilityFunctions.containsKey(TEST_EXEC_FUNCTION)) {
+                ReflectConfigClass testTestExecuteGeneratedRefConfClz = new ReflectConfigClass(
+                        testSuiteMap.get(moduleName).getTestUtilityFunctions().get(TEST_EXEC_FUNCTION));
+                testTestExecuteGeneratedRefConfClz.addReflectConfigClassMethod(
+                        new ReflectConfigClassMethod(
```

Comment:
Since ReflectConfigClassMethod is made with constants we can make whole object as a constant so we do not need to create ReflectConfigClassMethod every time